acceptance/acceptance_test.go

```diff
@@ -639,13 +639,23 @@ func testAcceptance(
 				})
 
 				when("default builder is set", func() {
+					var creatorSupported bool
+
 					it.Before(func() {
 						h.Run(t, subjectPack("set-default-builder", builderName))
+
+						// Technically the creator is supported as of platform API version 0.3 (lifecycle version 0.7.0+) but earlier versions
+						// have bugs that make using the creator problematic.
+						lifecycleSupportsCreator := !lifecycleDescriptor.Info.Version.LessThan(semver.MustParse("0.7.5"))
```

Comment:
I thought that we're good from [lifecycle version `0.7.4`](https://github.com/buildpacks/lifecycle/releases/tag/v0.7.4).
I know that version [`0.7.5`](https://github.com/buildpacks/lifecycle/releases/tag/v0.7.5) fixes anonymous auth when analyzer and exporter are run as root, but I thought that anyway we're ok with `0.7.4`.
Am I missing something?

Reply:
You are correct :)